xcode/Subconscious/Shared/Components/Detail/MemoEditorDetailMetaSheetView.swift

```diff
@@ -8,10 +8,10 @@
 import SwiftUI
 import ObservableStore
 
-struct DetailMetaSheet: View {
+struct MemoEditorDetailMetaSheetView: View {
```

Comment:
Change name of component to differentiate between e.g. viewer meta sheets vs editor meta sheets, which have different features.